lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -920,22 +920,31 @@ class MissingImportFixItCache {
   }
 };
 
-static void
+static bool
 diagnoseMissingImportsForMember(const ValueDecl *decl,
                                 SmallVectorImpl<ModuleDecl *> &modulesToImport,
                                 SourceFile *sf, SourceLoc loc) {
   auto &ctx = sf->getASTContext();
   auto count = modulesToImport.size();
   ASSERT(count > 0);
 
+  bool isMigrating =
+      ctx.LangOpts.getFeatureState(Feature::MemberImportVisibility)
+          .isEnabledForMigration();
```

Comment:
@AnthonyLatsis is it currently possible to coalesce the fix-it from multiple different diagnostics (possibilities here are - if I understand correctly - the same id but different arguments, locations, different ids), or we'd just end up adding multiple `import <<module>>` from that?

Reply:
Yes, something like that, let's wait for @AnthonyLatsis to confirm but I have a suspicion that we might not be able to coalesce the fix-its...

Reply:
I've implemented the suggested approach where in migration mode one warning is emitted per module that needs to be imported. Even if the migration infrastructure can handle the duplicate diagnostics that were emitted previously, I think this is going to yield the best developer experience.

Reply:
LGTM, thank you!